src/Avalonia.Base/Media/ImmutableGeometry.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Avalonia.Platform;
+
+namespace Avalonia.Media;
+
+internal class ImmutableGeometry : Geometry
```

Comment:
Introduced this to store the returned widened geometry. Because the geometry can't change once calculated, it's "immutable" but this naming may be confusing as we have other immutable public drawing primitives in `Avalonia.Media.Immutable`?

Reply:
`PlatformGeometry`?